06-best-practices/code/infrastructure/README.md

```diff
@@ -0,0 +1,75 @@
+## IaC w/ Terraform
+
+### Project infrastructure modules:
```

Comment:
it seems to mostly repeat the previous readme file. maybe remove the duplicated parts?